boards.txt

```diff
@@ -41058,43 +41058,158 @@ alfredo-nou3.menu.EraseFlash.all.upload.erase_cmd=-e
 
 ##############################################################
 codecell.name=CodeCell
-codecell.vid.0=0x303a
-codecell.pid.0=0x1002
-codecell.upload_port.0.vid=0x303a
-codecell.upload_port.0.pid=0x1002
 
 codecell.bootloader.tool=esptool_py
+codecell.bootloader.tool.default=esptool_py
+
 codecell.upload.tool=esptool_py
-codecell.upload.maximum_size=4194304
+codecell.upload.tool.default=esptool_py
+codecell.upload.tool.network=esp_ota
+
+codecell.upload.maximum_size=1310720
 codecell.upload.maximum_data_size=327680
+codecell.upload.flags=
+codecell.upload.extra_flags=
 codecell.upload.use_1200bps_touch=false
 codecell.upload.wait_for_upload_port=false
 
-codecell.upload.speed=921600
 codecell.serial.disableDTR=false
 codecell.serial.disableRTS=false
 
 codecell.build.tarch=riscv32
 codecell.build.target=esp
 codecell.build.mcu=esp32c3
 codecell.build.core=esp32
-codecell.build.variant=codecell
+codecell.build.variant=esp32c3
 codecell.build.board=ESP32C3_DEV
```

Comment:
This probably needs to be changed to reflect the board name

Reply:
board name is codecell - will it use codecell varriant pins_arduino.h?

Reply:
in that case old variant was OK and `build.board` should become `CODECELL`